lib/db.js

```diff
@@ -3,6 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 var P = require('bluebird')
+var extend = require('util')._extend;
```

Comment:
I hear no objections, even if it is a little different. This being db-mem (rather than db-mysql) - I'm okay with this. Raise your objections now if anyone has any! :)